dashboard/modules/job/job_manager.py

```diff
@@ -569,6 +569,10 @@ async def submit_job(
             start_time=int(time.time() * 1000),
             metadata=metadata,
             runtime_env=runtime_env,
+            driver_agent_http_address="http://"
+            f"{ray.worker.global_worker.node.node_ip_address}:"
+            f"{ray.worker.global_worker.node.dashboard_agent_listen_port}",
+            driver_node_id=ray.worker.global_worker.current_node_id.hex(),
```

Comment:
Should these parameters depend on `_driver_on_current_node`?

Reply:
sorry, I forget to remove `_driver_on_current_node`, and driver_agent_http_address should set on SupervisorActor, not in the jobmanager.